packages/components/src/components/Address/Address.test.tsx

```diff
@@ -1,11 +1,45 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { jest } from '@jest/globals';
+import { render, fireEvent, waitFor } from '@testing-library/react';
 
 import { Address } from './Address';
 
+Object.assign(navigator, {
+  clipboard: {
+    writeText: () => {},
+  },
+});
```

Comment:
Can you add a comment here that explains what this code does? (More importantly, why it was added?)

Reply:
fixed in [c12b761958d8a86a19c1220aa4b97d2aeff9ebaf](https://github.com/Developer-DAO/web3-ui/pull/105/commits/f14a7fb908e032e913e8dee684793cdb510c1b6d)